tests/serac_solid_reuse_mesh.cpp

```diff
@@ -129,12 +125,12 @@ TEST(solid_solver, reuse_mesh)
   double dt = 1.0;
   solid_solver_3.advanceTimestep(dt);
 
-  double u_norm_3 = solid_solver_3.displacement().gridFunc().ComputeLpError(2.0, zerovec);
+  double u_norm_3 = solid_solver_3.displacement().norm();
   EXPECT_NEAR(0.0, u_norm_1 - u_norm_3, 0.001);
```

Comment:
we should probably be checking ||u_i - u_j|| rather than ||u_i|| - ||u_j||  in these tests